wiki/Game_modifier/Fade_In/id.md

```diff
@@ -0,0 +1,42 @@
+---
+stub: true
+tags:
+  - fade in
+  - fade-in
+  - FI
+  - mod
+  - game modifier
+---
+
+# Fade In (mod)
+
+![Ikon mod FI](/wiki/shared/mods/FI.png "Ikon mod Fade In (FI)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*
+
+*Berbeda konteks dengan [Hidden (mod)](/wiki/Game_modifier/Hidden) atau [Flashlight (mod)](/wiki/Game_modifier/Flashlight).*
+
+## Tentang
+
+- Singkatan:  FI
+- Jenis: Peningkat kesulitan
+- Pengali Skor: 1.00x
+- Tombol shortcut: `F`
+- Mode permainan yang cocok: ![][o!m]
+- Variasi: [Hidden](/wiki/Game_modifier/Hidden)
+
+## Deskripsi
+
+mod **Fade In** adalah sebuah [game modifier](/wiki/Game_modifier) untuk [osu!mania](/wiki/Game_mode/osu!mania) yang meningkatkan kesulitan pada sebuah [beatmap](/wiki/Beatmaps) dengan cara menghilangkan note-note ketika mereka mendekati bar penilaian. Itulah asal muasal namanya.
+
+kecepatan pemudaran note tersebut didasarkan pada kombo, sehingga kombo yang lebih tinggi menyebabkan note semakin memudar tidak terlihat dalam jangka waktu yang lebih lama. Yaitu note akan memudar dan semakin memudar karena semakin banyaknya kombo yang diraih oleh pemain.
```

Comment:
`kecepatan` -> `Kecepatan`

also you don't really start a sentence with `Yaitu` tho, sth like this would be more preferable :

`Seiring waktu, not-not yang ada akan semakin memudar dengan semakin banyaknya kombo yang diraih oleh pemain.`